quiner/jwt-algorithm-bypass.yaml

```diff
@@ -0,0 +1,26 @@
+rules:
+- id: jwt-algorithm-bypass
+  languages:
+  - python
+  pattern: |
+    jwt.decode(..., "...", ...)
```

Comment:
This needs to be `jwt.decode($METAVARIABLE, "...", ...)` otherwise it'll match any string inside the decode function